dev-tools.sh

```diff
@@ -34,5 +34,6 @@
 #
 # Uncomment and set the Dev-Tools's commit value and commit this change.
 # export GH_COMMIT=COMMIT_SHA
+export GH_COMMIT=a93e9347eaedfebe31970929c16d4d8b17daee3b
```

Comment:
this needs to be removed once the dev-tools get a release with this changes - https://github.com/dpc-sdp/dev-tools/pull/71